### DISCUSSION TITLE
Does Parquet statistics check correctness?

### DISCUSSION THREAD

**Discussion by jp0317** [apache/arrow-rs#6405] [General]
In C++ implementation there's a [check](https://github.com/apache/arrow/blob/main/cpp/src/parquet/metadata.cc#L303) to effectively disable incorrect statistics, but it seems there's no equivalent in rust.  Is this intended?

**Comment by alamb** [comment]
I don't think we have thought carefully about what "incorrect statistics" means -- I am not sure what the rationale for the C/C++ code is, and thus I don't know if it would be a good idea to mirror the same behavior in Rust

**Comment by Jefffrey** [comment]
Feel free to raise an issue if this is a feature request for the arrow-rs implementation (with details such as what correctness means, etc.)
